Documentation/TO DO.md

```diff
@@ -1,25 +1,29 @@
 TO DO
 
+- Fortsätt bryta ut Menu.WantToMoveTwoPiecesFromYard() ur if-satserna.
+- Skriva tester för den nya logiken i gameEngine.(börjat)
+- Fixa menymetoderna(börjat)
 - Vi vill använda enum colors istället för att ha den som en sträng i klassen piece
-- Planera för vilka tester vi ska göra först
 - Fundera över AI-spelare och flowchart
```

Comment:
Gillar tänket med en To-do lista med avklarade moment! Något vi började med först idag